### PR TITLE
cllocation-placemarks

### PR DESCRIPTION
**Pull request by trevorphillips**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.0.
- [x] New extensions support iOS 10.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

File: Sources/SwifterSwift/CoreLocation/CLLocationExtensions.swift [hunk 1]
```diff
@@ -70,6 +69,21 @@ public extension CLLocation {
         return (degrees+360).truncatingRemainder(dividingBy: 360)
     }
 
+    /// SwifterSwift: Retrieve the first placemark of a location.
+    ///
+    /// - Parameter completion: completion handler to run after reverseGeocodeLocation finishes.
+    /// - Returns: An array of  placemarks for a given location or nil if it didn't find any.
```
Comment (LucianoPAlmeida) [inline Sources/SwifterSwift/CoreLocation/CLLocationExtensions.swift]: The function doesn't return, maybe it is better to describe this on the completion paramter result :)

**Comment by LucianoPAlmeida** [comment]
@guykogus I pretty much agree with all the points you've mention, and also there is the fact that this pretty much hides any underlaying error that may occur, so would not be possible to error handling correctly ...

**Comment by trevorphillips** [comment]
@guykogus @LucianoPAlmeida I believe I solved the user not being able to see the error that occurs but not the retention of the CLGeocoder object.

**Comment by LucianoPAlmeida** [comment]
> @guykogus @LucianoPAlmeida I believe I solved the user not being able to see the error that occurs but not the retention of the CLGeocoder object.

But I agree with @guykogus in the sense that it is not worth the abstraction ... meaning that we create a method that just calls another method with no difference on parameters and just forwards the response ... and in fact, as Guy mentioned, having a downside. So I agree that this would not be a good addition :)

**Pull request closed without merging**